### PR TITLE
[DDIDO-186] Added non-prod and Fixed Prod Routes

### PR DESCRIPTION
**Pull request by Yuuupie**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/DDIDO-186

### Changed

1.  Added routes for master and develop.
2.  Made prod route go under prod environment instead of master.

### CODE REVIEW

**Review by tim-yao — APPROVED** [review]

File: .lagoon.yml [hunk 1]
```diff
@@ -11,16 +11,28 @@ tasks:
...
+  production:
     routes:
       - storybook:
         - "www.ripple.sdp.vic.gov.au":
             tls-acme: 'false'
             insecure: Allow
             hsts: max-age=31536000
-        - "ripple.sdp.vic.gov.au":
+    monitoring_urls:
+      - "www.ripple.sdp.vic.gov.au"
```
Comment (FleetAdmiralButter) [inline .lagoon.yml:22]: Do we need to have monitoring set up for storybook?

**Review by FleetAdmiralButter — APPROVED** [review]
Looks like Lagoon successfully parsed this in the build.

File: .lagoon.yml (hunk 1 above)
Comment (Yuuupie) [reply .lagoon.yml:22]: Well storybook is the service that the routes point to so yes, but I'm guessing you might mean do we need to monitor the app route then? For that I don't know.

Comment (FleetAdmiralButter) [reply .lagoon.yml:22]: Sorry - I meant as in since Storybook isn't really a consumer facing site.

Comment (Yuuupie) [reply .lagoon.yml:22]: I get you now. @kurtfoster Any thoughts?

Comment (kurtfoster) [reply .lagoon.yml:22]: @Yuuupie / @FleetAdmiralButter , I don't think we need to monitor storybook.

**Review by tim-yao — APPROVED** [review]

**Review by kurtfoster — APPROVED** [review]

**Pull request merged**